systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PersistentVolumeClaimUtils.java

```diff
@@ -47,4 +50,22 @@ public static void waitUntilPVCAnnotationChange(Map<String, String> newAnnotatio
             });
         LOGGER.info("PVC annotation has changed {}", newAnnotation.toString());
     }
+
+    public static void waitUntilPVCDeletion(String clusterName) {
+        LOGGER.info("Waiting till PVC deletion for cluster {}", clusterName);
+        TestUtils.waitFor("Waiting till PVC will be deleted {}", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_STATUS_TIMEOUT,
+            () -> {
+                List<PersistentVolumeClaim> pvcList = kubeClient().listPersistentVolumeClaims().stream().filter(pvc -> pvc.getMetadata().getName().contains(clusterName)).collect(Collectors.toList());
+                if (pvcList.isEmpty()) {
+                    return true;
+                } else {
+                    for (PersistentVolumeClaim pvc : pvcList) {
+                        LOGGER.warn("PVC {} is not deleted yet! Triggering force delete by cmd client!", pvc.getMetadata().getName());
+                        cmdKubeClient().deleteByName("pvc", pvc.getMetadata().getName());
```

Comment:
a question ... we have `deleteClaim` property on the storage which is actually for deleting PVCs when the cluster is deployed. Should you set this flag for your deployed Kafka cluster and then waiting just for deletion instead of forcing it?

Reply:
It's a valid point. I was thinking that we have `deleteClaim: true` by default but by default it's set to `fase`. I will change this as well. However, this check which I added is just double check that everything is deleted before new test starts and in case if's still up, we force the deletion. We are doing same with STs, Deployment, etc.

Reply:
Yes we have separated tests for that. I will double check if they are working as expected with these changes just to avoid need to create another fix PR.

Reply:
Tests related to deleteClaim options are fine so I am going to mark this one as ready for merge.